portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,24 @@
+@component-name = "portal"
+definition {
+
+	property testray.main.component.name = "Navigation";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	@description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
```

Comment:
Hey @Claudioss23, what do you think about changing the Portal Global Menu to -> Access `Applications Menu`? 
Because this is what this menu is called inside DXP. :slightly_smiling_face: thank you!

Reply:
Alright, makes sense. I'll fix that right up!